### PR TITLE
#8: test commit

### PR DESCRIPTION
**Pull request by danilkuznetsov**
connect #8 Test update for readme file

### CODE REVIEW

**Comment by danilkuznetsov** [comment]
@vkuchyn need review

**Comment by vkuchyn** [comment]
@dtantsev pls review

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# microservices
```
Comment (vkuchyn) [inline README.md]: @danilkuznetsov why so?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+Microservices for juja online training cources
```
Comment (vkuchyn) [inline README.md]: @danilkuznetsov better to use Factory

**Comment by vkuchyn** [comment]
@danilkuznetsov view my recommendations pls

**Pull request closed without merging**